u8timeseries/models/rnn_model.py

```diff
@@ -30,8 +34,55 @@ def _get_runs_folder(work_dir, model_name):
     return os.path.join(work_dir, RUNS_FOLDER, model_name)
 
 
+class _TimeSeriesDataset1D(Dataset):
```

Comment:
Any specific reason this was moved here? I guess if this is our only torch-based model (which it currently is) it's a good place. But it might be useful to move this into utils for future torch-based models to have access to. What do you think?

Reply:
The main reason is that I think from now on we should consider the submodule `utils` as something "public-facing". The users of u8timeseries are unlikely to care about the Pytorch dataset, as it is mosty meant only to be used internally by the RNN model. I think if we later have more Pytorch models requiring such Datasets, we should maybe factor this one out, but not in the public-facing `utils` submodue. Makes sense for you?

Reply:
Got it! Maybe we can add it in a separate file in the models module but not include it in `__init__.py`? I guess we still have time to think about it.

Reply:
Yes that could be a good option, however I think we should leave it for a next PR